duels-plugin/src/main/resources/lang.yml

```
@@ -700,4 +739,12 @@ RANK:
   not-found: '{PREFIX} &cRank not found.'
-  reloaded: '{PREFIX} &aRank configuration has been reloaded.'
\ No newline at end of file
+  reloaded: '{PREFIX} &aRank configuration has been reloaded.'
+  system-disabled: '{PREFIX} &cRank system is currently disabled.'
+  data-load-error: '{PREFIX} &cUnable to load your user data.'
+  info:
+    header: '&9&l=== Rank Information ==='
+    rank: '&7Rank: %rank%'
+    description: '&7Description: %description%'
+    progress-to-next: '&7Progress to &e%next_rank%&7: &e%progress%&7% (&e%elo_needed%&7 ELO needed)'
+    total-elo: '&7Total ELO: &e%elo%'
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)